channels/chan_echolink.c

```
@@ -3709,9 +3982,8 @@ static void *el_reader(void *data)
 					}
 				} else {
 					if (is_rtcp_bye((unsigned char *) buf, recvlen)) {
-						if (find_delete(&instp->el_node_test)) {
-							ast_verb(4, "Disconnect from IP address %s, Callsign %s.\n", instp->el_node_test.ip,
-								instp->el_node_test.call);
+						if (find_delete(&node_lookup, instp)) {
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)